### ISSUE TITLE
fail to use GN, any suggestion is welcome.

### ISSUE THREAD

**Issue by FishLikeApple** [open-mmlab/mmdetection#1111]
Hi. I'm training a model for object detection. The model originally used syncBN, but I only have access to one GPU and the batch size in training is very small (1 or 2), so I changed syncBN to GN (without considering BN due to very small batch), but the model then just output nothing (no bbox).
After some discovery, I found that the model's output seemed independent of input. At first, the output was all with the same class with the same confidence (0.06-0.09), as loss decreased,  output became nothing (no bbox).
What's interesting is that when I tested the model with BN, it worked fine (at least the output is something related to input).
I'm here to find some clue. Any suggestion is welcome.

**Comment by FishLikeApple** [comment]
Can I use syncBN with only one GPU? I'd like to not change original setting before full understanding.

**Comment by FishLikeApple** [comment]
How should I change the learning rate when changing syncBN to GB?

**Comment by FishLikeApple** [comment]
Original: norm_cfg = dict(type='syncBN', requires_grad=True)
Now:      norm_cfg = dict(type='GN', num_groups=32, requires_grad=True).

**Comment by FishLikeApple** [comment]
I'm sorry, BN seems also not fine, I will try to find out where the problem is.

**Comment by FishLikeApple** [comment]
I think the model’s structure is fine due to that the model works well with the official checkpoint file, but I can't train a working model on a new dataset. 
I have checked the input to the model, the input data is fine. now I'm checking forward and backward steps. I hope I can find the bug.
